frontend/src/atoms.tsx

```diff
@@ -243,7 +243,7 @@ export function LoadData(props: { children?: ReactNode }) {
         addSetter(ChannelApiVersion, ChannelKind, setChannelsState)
         addSetter(PlacementRuleApiVersion, PlacementRuleKind, setPlacementRulesState)
         addSetter(SubscriptionApiVersion, SubscriptionKind, setSubscriptionsState)
-        addSetter('apps.open-cluster-management.io/v1alpha1', 'GitOpsCluster', setGitOpsClustersState)
+        addSetter('apps.open-cluster-management.io/v1beta1', 'GitOpsCluster', setGitOpsClustersState)
```

Comment:
@chenz4027 Can you import the apiVersion and kind from gitops-cluster.ts just like the other resources? This way we only need to update the version in one place when it changes

Reply:
Sure I'll make the change